src/dsp_tools/commands/xml_validate/xml_validate.py

```diff
@@ -39,10 +40,11 @@ def xml_validate(filepath: Path, api_url: str, dev_route: bool) -> bool:  # noqa
     _inform_about_experimental_feature()
     data_rdf, shortcode = _get_data_info_from_file(filepath, api_url)
     onto_con = OntologyConnection(api_url, shortcode)
-    ontologies = _get_project_ontos(onto_con)
-    data_graph = data_rdf.make_graph() + ontologies
+    ontologies, shapes = _get_shacl(onto_con)
+    data_graph = data_rdf.make_graph()
+    # data_graph += ontologies
```

Comment:
This is a cheap fix and the basis for our meeting on monday, if we exclude the ontology from the val.
Currently our shapes are closed, so that no properties can be used that are not ennumerated. What I did not consider, this is a problem when a sub-class has properties that it's super-class does not have. At the moment not having inheritance solves the problem. The newly crated correct test data passes. This is not an option as soon as we validate the range of the link properties. As this is a future feature I decided to implement this.